pages/animals/index.tsx

```diff
@@ -22,10 +22,6 @@ function Animals() {
 
   const [clientDialog, setClientDialog] = useState(false);
 
-  useEffect(() => {
```

Comment:
Why was this removed?

Reply:
I don't have the context for it. Jeff removed this a month ago. Also removed the getTickets() function from the ClientService, which wasn't using the DB anyway, so perhaps it was just him cleaning up code. 

We'll get this and other pages working and fetching from the DB in the upcoming sprints.

Reply:
Ok gotcha